sam/bamWrite_test.go

```diff
@@ -43,7 +40,7 @@ func testFile(file string) error {
 		return err
 	}
 
-	f, err := ioutil.TempFile("testdata", "tmp*.bam")
```

Comment:
ioutil is considered deprecated
- https://medium.com/@ButcherWithSmile/replacing-golangs-ioutil-functions-and-methods-ec9dca2c7e59